docs/specification.md

```diff
@@ -490,19 +441,18 @@ Parameter (in Michelson)
 - This entrypoint MUST follow the FA2 requirements.
 
 - Permission logic requirements specific to this contract are described in the ["FA2 Specifics"](#fa2-specifics) chapter.
-Also see [this issue](https://gitlab.com/tzip/tzip/-/issues/16).
+Each `owner` must be equal to `SENDER`.
 
 - Since the contract supports only a single token type, all `token_id` values MUST be 0.
   They are passed because FA2 requires that.
-  So there are at most three possible valid values of `operator_tokens` type:
+  So there are three possible valid values of `operator_tokens` type:
```

Comment:
Check recent updates to FA2 standard. Operators are simplified and do not have per token granularity anymore